src/platform/user/authorization/components/RequiredLoginView.jsx

```diff
@@ -14,6 +14,14 @@ const nextQuery = { next: window.location.pathname };
 const signInUrl = appendQuery('/', nextQuery);
 const verifyUrl = appendQuery('/verify', nextQuery);
 
+const RequiredLoginLoader = () => {
```

Comment:
Attention VSP reviewers: This file is the only platform-level file that needed to be updated. Well, this and the corresponding test file :)